Java/cryptography/playfairCipher.java

```diff
@@ -0,0 +1,123 @@
+import java.util.*;
+class Main
+{  
+   static int tellRow(char a[][] , char ch)
```

Comment:
Use proper comments to explain what each function does. And use any [Java beautifier](https://codebeautify.org/javaviewer) to format your code.